CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [Unreleased]
+
+## Added
+- Added CVE-2019-3016 mitigation information in [Production Host Setup](docs/prod-host-setup.md)
```

Comment:
One more nit: the information is updated instead of added.

Reply:
My bad - wrong CVE :)

Reply:
I'm confused, wasn't this released this week?